### PR TITLE
Update to angular 13

### PR DESCRIPTION
**Pull request by mbarbet**

### CODE REVIEW

**Review by MohamedHamouGisaia — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -5,49 +5,67 @@
...
     "@biesbjerg/ngx-translate-extract-marker": "1.0.0",
-    "@types/jasmine": "2.8.8",
+    "@types/jasmine": "~3.10.0",
     "@types/jasminewd2": "~2.0.2",
-    "@types/node": "^8.10.36",
-    "codelyzer": "^5.1.2",
-    "jasmine-core": "~2.99.1",
+    "@types/node": "^12.20.38",
+    "@typescript-eslint/eslint-plugin": "5.3.0",
+    "@typescript-eslint/eslint-plugin-tslint": "^5.6.0",
```
Comment (sebbousquet) [inline package.json:52]: You can remove this plugin and all config link to it

File: package.json [hunk 2]
```diff
@@ -5,49 +5,67 @@
...
-    "protractor": "~5.4.0",
-    "ts-node": "~7.0.0",
+    "karma": "^6.3.4",
+    "karma-chrome-launcher": "~3.1.0",
+    "karma-jasmine": "~4.0.0",
+    "karma-jasmine-html-reporter": "~1.7.0",
+    "license-check-and-add": "^3.0.4",
+    "patch-package": "^6.4.7",
+    "protractor": "~7.0.0",
     "tslint": "~5.16.0",
```
Comment (sebbousquet) [inline package.json:65]: You can also remove this dependency

File: package.json (hunk 1 above)
Comment (sebbousquet) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (sebbousquet) [reply package.json]: Done

**Review by sebbousquet — APPROVED** [review]

**Pull request merged**